services/grafana/client.go

```diff
@@ -534,7 +535,7 @@ func (c *Client) GetCurrentUserAccessToken(ctx context.Context) (string, error)
 	if err := c.do(ctx, http.MethodGet, "/percona-api/user/oauth-token", "", headers, nil, &user); err != nil {
 		var e *clientError
 		if errors.As(err, &e) && e.ErrorMessage == "Failed to get token" && e.Code == http.StatusInternalServerError {
-			return "", errors.New("failed to get token")
+			return "", services.ErrFailedToGetToken
```

Comment:
this was a mistake in previously committed code which led to the platform endpoints returning 500 error instead of the actual error